apps/asap-server/src/controllers/teams.ts

```diff
@@ -44,7 +44,7 @@ flatData {
     url
   }
 }
-referencingUsersContents {
+referencingUsersContents(filter: "data/onboarded/iv eq true") {
```

Comment:
with how we do tests in asap-server there is currently no way to write a sensible test case for this type of change (ie query change)